tensorflow/compiler/tf2tensorrt/convert/convert_nodes.h

```diff
@@ -46,6 +46,19 @@ namespace tensorrt {
 namespace convert {
 using ::stream_executor::port::StatusOr;
 
+#define TFTRT_INTERNAL_ERROR_AT_NODE(node)                           \
+  do {                                                               \
+    return errors::Internal("TFTRT::", __FUNCTION__, ":", __LINE__,  \
+                            " failed to add TRT layer, at: ", node); \
+  } while (0)
+
+#define TFTRT_RETURN_ERROR_IF_NULLPTR(ptr, node) \
+  do {                                           \
+    if (ptr == nullptr) {                        \
+      TFTRT_INTERNAL_ERROR_AT_NODE(node);        \
+    }                                            \
+  } while (0)
+
```

Comment:
I can't tell why you want to move this from *.cc to *.h. Was thinking you need to use them in another file, but couldn't find such new usage. Can you explain?

Reply:
I use it here https://github.com/tensorflow/tensorflow/pull/54170/files#diff-1a522f9483979968cdda7c2f25f287b64d89bb09d82bd0b356699076bcd99b9eR122